docs/what_is_securedrop.rst

```diff
@@ -180,5 +171,5 @@ to check out the :ref:`best practices <Landing Page>` for your
 SecureDrop *Landing Page* and our guide to
 :doc:`promoting your SecureDrop instance <getting_the_most_out_of_securedrop>`.
 
-.. |SecureDrop architecture overview diagram| image:: ./diagrams/SecureDrop.png
+.. |SecureDrop architecture highlevel overview diagram| image:: ./diagrams/securedrop_overview_highlevel.png
```

Comment:
This diagram does not include the admin role, but it follows a paragraph that describes admins as one of the four components. I think that's potentially confusing for a reader trying to visually map the preceding paragraph to the diagram.

I would place this a bit further down (so that it's not seen as a direct visual representation of the preceding paragraph) and annotate, e.g., as follows:

> This diagram shows the process of using SecureDrop from the perspective of sources and journalists. Sources and journalists connect to SecureDrop using the Tor network (represented by the onion symbol). The SecureDrop software is running on premises on dedicated infrastructure.
>
> A source (bottom left) uploads a submission to the news organization using Tor Browser. A journalist connects to SecureDrop using their Journalist Workstation (booted from a USB drive) and physically transfers files to the air-gapped Secure Viewing Station, a machine that is never connected to the Internet. On this computer, they can process or print documents, or export them to a dedicated device.

Reply:
I agree, I made the change as requested using the annotation you provided.